suite-native/atoms/src/Sheet/BottomSheetContainer.tsx

```diff
@@ -1,17 +1,23 @@
 import { ReactNode } from 'react';
-import { KeyboardAvoidingView, Modal as RNModal, Platform } from 'react-native';
+import { KeyboardAvoidingView, Modal as RNModal, Platform, Dimensions } from 'react-native';
 import { gestureHandlerRootHOC } from 'react-native-gesture-handler';
 
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 
+const SCREEN_SIZE = Dimensions.get('window');
+
 type SheetProps = {
     children: ReactNode;
     isVisible: boolean;
     onClose: () => void;
     ExtraProvider?: React.ComponentType;
 };
 
-const ContentWrapperStyle = prepareNativeStyle(_ => ({ flex: 1 }));
+const ContentWrapperStyle = prepareNativeStyle(_ => ({
+    flex: 1,
+    width: SCREEN_SIZE.width,
+    height: SCREEN_SIZE.height,
```

Comment:
What about using `EnvUtils` instead?

Reply:
Yes, and the name screen_size is confusing since it is window size...

Reply:
Yes, that is why you should use it...

Reply:
We should probably then replace all usage of `Dimensions.get('screen')` and replace it by this through the app, because that is how it is being used for now.

Reply:
changed in f379a0f34e

Reply:
Yes, we should... Thanks for this change!